docs/what-is-bitcoin.md

```diff
@@ -0,0 +1,71 @@
+## Overview
+
+Bitcoin is a peer-to-peer digital currency and network. Bitcoins can be sent and received through
+the internet, and aren't centrally controlled by anyone. Unlike using a bank, this means nobody
+can seize, block or take control of your money. Instead, Bitcoin is run and verified collectively by
+its users, as a peer-to-peer network. Let's get started and explain the history behind Bitcoin
+and how it works.
+
+## Background and transactions
+
+Bitcoin was first described in the Bitcoin whitepaper, by Satoshi Nakamoto, its first developer.
+Before Bitcoin, digital currencies and payment systems all relied on trust, by either trusting a
+company, or an individual. This made it easy for these trusted persons to block people's money,
+or succumb to Government intervention. Satoshi figured out a way to overcome that, by inventing
+a data structure called the blockchain, which is distributed between all users in a peer-to-peer network.
```

Comment:
LINE 15: "a data structure known as Blockchain. This blockchain is a publicly distributed ledger that is maintained and securely updated by the Bitcoin Network."